scene/gui/graph_node.cpp

```diff
@@ -664,7 +664,7 @@ Size2 GraphNode::get_minimum_size() const {
 
 	for (int i = 0; i < get_child_count(); i++) {
 		Control *c = Object::cast_to<Control>(get_child(i));
-		if (!c) {
+		if (!c || !c->is_visible_in_tree()) {
```

Comment:
What I was saying is that we don't need to check if it's visible in tree, we only need to check if it's visible itself.

```suggestion
		if (!c || !c->is_visible()) {
```

That's enough here, and would in fact be in line with what other containers are doing.

Reply:
Ah, sorry, I misunderstood. I have changed as you suggested and verified that this still works in my use case